nupic/algorithms/CLAClassifier.py

```diff
@@ -553,13 +548,15 @@ def read(cls, proto):
     classifier.alpha = proto.alpha
     classifier.actValueAlpha = proto.actValueAlpha
     classifier._learnIteration = proto.learnIteration
-    classifier._recordNumMinusLearnIteration = proto.recordNumMinusLearnIteration
+    classifier._recordNumMinusLearnIteration = \
```

Comment:
Don't use backslashes. Use parens for implicit line continuations. Details here:
https://www.python.org/dev/peps/pep-0008/#maximum-line-length